### ISSUE TITLE
edge-cloudflare function `getPayloadFromKV` is accessing payload incorrectly

### ISSUE THREAD

**Issue by andrew-gamurs** [growthbook/growthbook-proxy#66]
We updated our Cloudflare Growthbook integration to 1.0.0 to utilise the native integration with CF, but when we replaced the old `loadFeatures` function with `init` we saw that no features were being loaded. With some debugging we noticed that the `getPayloadFromKV` function seems to be accessing the payload incorrectly.

The payload from the webhook has the following structure, where the actual `payload` is stringified JSON under `data.payload`:

```
{
  "type": "payload.changed",
  "timestamp": "2024-07-09T08:24:38.074Z",
  "data": {
    "payload": "{\"features\": {...}, \"dateUpdated\": \"2024-07-09T08:24:38.022Z\"}"
  }
}
```

But the function is taking the webhook payload as the GB payload itself. We patched our version of `@growthbook/edge-cloudflare` so the function looks like the following:

```
export async function getPayloadFromKV(
  env: Env,
  namespace: string = "KV_GB_PAYLOAD",
  key: string = "gb_payload",
) {
  if (env?.[namespace]) {
    const KV: KVNamespace = env[namespace];
    const value = (await KV.get(key, {type: "json"})) || undefined;
    let payload = undefined;
    if (value && value?.data?.payload) {
      try {
        payload = JSON.parse(value.data.payload) as FeatureApiResponse;
      } catch (e) {
        console.warn("Unable to parse payload", e);
      }
    }
    return payload;
  }
}
```

And now the `init` function loads the features as expected.

**Comment by bryce-fitzsimons** [comment]
Hi, we recently patched this at the GB app level. Instead of having the CF worker look for the correct nested payload, we now allow SDK Connection > Webhooks to send in the proper format. See updated docs here:
https://docs.growthbook.io/lib/edge/cloudflare#configuring-a-sdk-webhook

TL;DR: Please set your webhook payload format to "Payload only"
<img width="831" alt="image" src="https://github.com/user-attachments/assets/e9f19470-3e8c-4500-8ea9-92fa5f9f3a6a">

Closing issue for now; feel free to re-open it though.
